addons/udes_stock/models/stock_location.py

```diff
@@ -428,6 +435,7 @@ def _process_inventory_adjustments(self, adjustments_request):
               'product_qty':  quantity,
               'location_id':  self.id,
               'package_id':   stock_info.package_id,
+              'u_result_parent_package_id': stock_info.parent_package_id,
```

Comment:
Do we have to set `u_package_parent_package_id` in case `stock_info.package_id` has it?

Reply:
Now that stock inventory lines include this field we have to pass it on creation otherwise it would remove it from the pallet. Also, if a stock check reveals the package is on a different pallet (in the same location), this will create an inventory adjustment to change it in the system